packages/uniforms/test/GraphQLBridge.js

```diff
@@ -36,22 +36,27 @@ describe('GraphQLBridge', () => {
 
     const schemaData = {
         author: {
+            initialValue: 'Jobs',
```

Comment:
Now I've noticed, that this is an invalid value because it's not a valid object of type `Author`.